### DISCUSSION TITLE
Lesson 5: Fail installing Brownie

### DISCUSSION THREAD

**Discussion by Sharkytec** [smartcontractkit/full-blockchain-solidity-course-py#1612] [Q&A] [answered]
I hit a deadlock at installing Brownie on Lesson 5.
I have successfully installed pipx and run the pipx ensurepath. 
But failed on "pipx install eth-brownie".
I got the below message from the terminal:
"pipx : The term 'pipx' is not recognized as the name of a cmdlet, function, script file, or operable 
program. Check the spelling of the name, or if a path was included, verify that the path is correct and 
try again.
At line:1 char:1
+ pipx install eth-brownie
+ ~~~~
    + CategoryInfo          : ObjectNotFound: (pipx:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException" 

Help!!

**Comment by cromewar** [comment, marked as answer]
Hello @Sharkytec 
The reason is windows does not how to handle those kind of packages unless you do some tricky stuff on the path variables.
Anyways using pipx is not worth it as Vscode will not recognize brownie anyway, so please use just regular pip to install eth-brownie.

Additionally, consider stop using just windows for this kind of development, WSL is the best option for this and you have a guide under "windows support" con the course index.
